controllers/index.controller.js

```diff
@@ -113,15 +113,18 @@ const RenderHomePage = async (req, res, next) => {
     mangass = await mangaParkObj.getMangaList(1);
     const mangaUpdateList = mangass.LatestManga;
 
-    res.render("index", {
-        title: "Mirai",
-        navList: navList,
-        updateList: updateList,
-        mangaUpdateList: mangaUpdateList,
-        newAnimeList: newAnimeList,
-        popularList: popularList,
-        ongoingList: ongoingList,
-    });
+    setTimeout(function(){
```

Comment:
Previously, since the render method for home page was working concurrently with the other web scraping functions, the home page was getting rendered before the fetching of ongoing anime list was complete.

Here, I have added a simple timeout of 3 seconds for the render home page function. This 3 second delay is enough for the web scraping functions to complete before the rendering happens - thus solving the issue.